soc/nordic/common/vpr/Kconfig.defconfig

```diff
@@ -1,12 +1,6 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-CPU_PATH := $(dt_nodelabel_path,cpu)
-CPU_ID := $(dt_node_reg_addr_int,$(CPU_PATH))
-
-config RV_BOOT_HART
-	default $(CPU_ID) if SOC_NRF54H20_ENGA_CPUPPR
-
```

Comment:
isn't this needed for FP1?

Reply:
PPR on FP1 won't boot if this is not set to 0 (which is the default value).